admin/fleetcommander/utils.py

```diff
@@ -115,3 +116,15 @@ def merge_settings(a, b):
         result[domain] = [index[key] for key in index]
 
     return result
+
+
+def get_free_port():
```

Comment:
Anyway I will need to check how to ask the port to soup because I need it for tests.